selectors/global_header.ts

```diff
@@ -3,9 +3,12 @@
 
 import {GlobalState} from 'types/store';
 import {getFeatureFlagValue} from 'mattermost-redux/selectors/entities/general';
+import * as UserAgent from '../utils/user_agent';
+import * as Utils from '../utils/utils';
 
 export function getGlobalHeaderEnabled(state: GlobalState): boolean {
     const featureFlagEnabled = getFeatureFlagValue(state, 'GlobalHeader') === 'true';
+    const isMobileView = UserAgent.isMobile() || Utils.isMobile();
```

Comment:
It looks like this unrelated code has appeared. Bad rebase?

Reply:
yep ... sadly! will remove